docs/FAQ.md

```diff
@@ -42,45 +45,56 @@ Here's a video explaining the git branch in short [![tutorial](https://img.youtu
 ### Github :octocat:
 
 **What to do next after creating a GitHub account?**
+
 - Fork the project from this [link](https://github.com/twindle-co/twindle)
 - Create the first PR.
 
 **Tips for your PR**
-- Use these [keywords](https://github.com/twindle-co/twindle/issues/167) in your PR description to make the issues linked (to the PR) close automatically once is merged.
 
-e.g. ```closes #102 #33 #14```
+- Use these [keywords](https://github.com/twindle-co/twindle/issues/167) in your PR description to make the issues linked (to the PR) close automatically once is merged.
 
+e.g. `closes #102 #33 #14`
 
 ### Google Meet 📹
 
 **Rules**
+
 - Mute yourself
 - Unmute only if you are speaking
 - Stay in a noise-free environment
 - If you are in a noisy environment don't speak
 
 **What will be the Google Meet timing?**
+
 - 5 pm IST
 
 **How to know what happened during that Google meet if I missed?**
+
 - Someone will take notes and share it with the team.
 - You can also watch the videos on this [list](https://youtu.be/i90UeTiEKQk) of the YT channel.
 
-
 **What is git branching in git?**
-* Git branches are alternative working directory system within git repository. 
-* In essence, they are the backbone of version control and code management feature provided by git.
-* Here's a video explaining the git branch in short:
 
-### Logo 🎨
+- Git branches are alternative working directory system within git repository.
+- In essence, they are the backbone of version control and code management feature provided by git.
+- Here's a video explaining the git branch in short:
+
+### Youtube 💻
+
+Find all tutorials and recorded meets on this [channel](https://www.youtube.com/channel/UCKxUmbHq5P5pd5IyUiZ8MHA)
+
+## Logo 🎨
 
 **Where to make the logo from?**
+
 - May use free platforms like Canva, Gimp, or Inkspace
 - Use photoshop (if you already have the software)
 
 **How to share the logo I made?**
-- Comment on the issue **[#4 "We need a logo"](https://github.com/twindle-co/twindle/issues/4)** raised by johnjacobkenny
```

Comment:
This was better 😅

Reply:
I made changes then discarded. This is just because of pulling from upstream.